ibraheemosule/skoolizy
# Fixed the verify account otp request flow and updated banner functionality

# Why this PR?
The sign up code flow causes two emails to be sent almost simultaneously and this causes some confusion on the user end

# What does this PR do?
- This PR ensures the otp email is sent based on the user request

# How to test your work?
- Create an account
- You should get a sign up successful email but not an otp email
- Otp email would be sent after the user clicks on send code

# Pages to see your changes?
- /auth/verify-account
